src/oss-detect-cryptography/DetectCryptographyTool.cs

```diff
@@ -92,9 +92,8 @@ private static async Task Main(string[] args)
                             {
                                 targetDirectoryName = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
                             }
-                            BaseProjectManager projectManager = ProjectManagerFactory.CreateBaseProjectManager(detectCryptographyTool.HttpClientFactory, targetDirectoryName);
 
-                            string? path = await projectManager.ExtractArchive("temp", File.ReadAllBytes(target));
+                            string? path = await ArchiveHelper.ExtractArchiveAsync(targetDirectoryName, "temp", File.OpenRead(target));
```

Comment:
Recommend changing this "Temp" to something more descriptive. The directory is already random and in the temp directory.